### ISSUE TITLE
workflow for identifying epidoc enhancements

### ISSUE THREAD

**Issue by balmas** [PerseusDL/tei-conversion-tools#39]
See https://github.com/PerseusDL/canonical/issues/155

**Comment by PonteIneptique** [comment]
In what way is it an issue for tei conversion tools ? Shouldn't it be an issue on Epidoc ?

**Comment by balmas** [comment]
yes, I'm working through this ... :) forget this approach

**Comment by PonteIneptique** [comment]
:')

**Comment by simonastoyanova** [comment]
This leads back to my question from a couple of weeks ago about building a
Perseus schema/ODD/whatever-you-want-to-call-it on top of the EpiDoc one,
since obviously a schema for inscriptions doesn't cover everything that a
print edition has. Bridget, what are you thinking for the castList in the
short term?

On 12 November 2015 at 14:09, Thibault Clérice notifications@github.com
wrote:

> :')
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/PerseusDL/tei-conversion-tools/issues/39#issuecomment-156098830
> .

## 

Simona Stoyanova
Research Associate

Digital Humanities
Department of Computer Science
University of Leipzig
Augustusplatz 10
04109 Leipzig, Germany

**Comment by balmas** [comment]
yes, we need to participate in that effort with @hcayless but I propose that we should gather the requirements rather than submitting piecemeal to the epidoc community.  

I've suggested we follow this workflow:

https://github.com/PerseusDL/tei-conversion-tools/wiki/EpiDoc-Conversions#what-if-a-mapping-is-missing-for-a-tei-tag

But now I have decided to reopen this issue, under a revised title, to make sure this workflow makes senses to everyone who is interested  and have tagged those who might be :)

@PonteIneptique @lcerrato @hcayless @simonastoyanova @TDBuck

**Comment by simonastoyanova** [comment]
Thanks for opening this again! And I agree.

On 12 November 2015 at 15:00, Bridget Almas notifications@github.com
wrote:

> Reopened #39 https://github.com/PerseusDL/tei-conversion-tools/issues/39
> .
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/PerseusDL/tei-conversion-tools/issues/39#event-462452783
> .

## 

Simona Stoyanova
Research Associate

Digital Humanities
Department of Computer Science
University of Leipzig
Augustusplatz 10
04109 Leipzig, Germany
